### PR TITLE
Bitgo api v2

### PR DESCRIPTION
**Pull request by RafaelTaranto**
Milestone: 7.3.x Crafty Chnemu

### CODE REVIEW

**Comment by RafaelTaranto** [comment]
[Update BitGo module to API v2](https://trello.com/c/6FaCLmf4/710-update-bitgo-module-to-api-v2)

File: lib/plugins/wallet/bitgo/bitgo.js [hunk 1]
```diff
@@ -36,21 +38,22 @@ function sendCoins (account, address, cryptoAtoms, cryptoCode) {
...
+      if (err.message === 'insufficient funds') throw new E.InsufficientFundsError()
       throw err
     })
 }
 
 function balance (account, cryptoCode) {
   return checkCryptoCode(cryptoCode)
     .then(() => getWallet(account))
-    .then(wallet => BN(wallet.wallet.spendableConfirmedBalance))
+    // TODO This was previously spendableConfirmedBalance, what happened to it on v2?
```
Comment (RafaelTaranto) [inline lib/plugins/wallet/bitgo/bitgo.js]: Shoud I use the minimum value between confirmedBalance and spendableBalance? Not sure what spendableConfirmedBalance means, and it not really documented on their api.

Comment (RafaelTaranto) [reply lib/plugins/wallet/bitgo/bitgo.js]: Some tests:

```
0.0013 cashout =>
balance: 0.09025480
confirmedBalance: 0.01075465
spendableBalance: 0.09025480

0.0013 cashout =>
balance: 0.08891092
confirmedBalance: 0
spendableBalance: 0.08891092

0.0005 cashin =>
balance: 0.08945823
confirmedBalance: 0
spendableBalance: 0.08891092
```

**Review by joshmh — CHANGES_REQUESTED** [review]

File: lib/plugins/wallet/bitgo/bitgo.js (hunk 1 above)
Comment (RafaelTaranto) [reply lib/plugins/wallet/bitgo/bitgo.js]: I'm converting before printout for readability.

**Pull request merged**